examples/text-generation/utils.py

```diff
@@ -367,7 +367,7 @@ def initialize_model(args, logger):
     set_seed(args.seed)
     get_repo_root(args.model_name_or_path, local_rank=args.local_rank, token=args.token)
     use_deepspeed = args.world_size > 0
-    if use_deepspeed or args.bf16 or args.fp8:
```

Comment:
This changes behavior than what we had before, is it tested on 7b? (seems like the only way to be false)